src/Simplic.FileStructure.UI/FileStructureControl.xaml

```diff
@@ -73,6 +73,11 @@
                                     <Image Height="16" Width="16" Source="{simplic:Icon Name=Edit_32x32}" />
                                 </telerik:RadMenuItem.Icon>
                             </telerik:RadMenuItem>
+                            <telerik:RadMenuItem Header="{simplic:Localization Key=filestructure_rename_directory}" Command="{Binding RenameDirectoryCommand}">
+                                <telerik:RadMenuItem.Icon>
+                                    <Image Height="16" Width="16" Source="{simplic:Icon Name=Rename_6779}" />

```

Comment:
We need to ensure our naming conventions here https://simplic.github.io/dev/gui/icons_filename.html